### ISSUE TITLE
Change Back, Forward, and Reload Shortcuts

### ISSUE THREAD

**Issue by AliyanH** [Maps4HTML/Web-Map-Custom-Element#766]
Currently The <ins>F</ins>orward and <ins>B</ins>ack history use the shortcut F and B respectively, when used from the contextmenu.

The Forward and Back history should instead use the Alt+Arrow Keys to navigate through the history (Overriding the default browser behavior similar to what's done with the [video element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)) when the map is focused. ~~when we reach the end of history, the Alt+Arrow Keys should not override the default browser behavior.~~ __The video element does not have this behavior?__

The <ins>R</ins>eload contextmenu button's shortcut should be changed from the R key to Ctrl+R When the map is focused, overriding the default browser behavior. 

This will help clear up the F Key, which can allow users to [toggle full screen](https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/201).

**Comment by prushforth** [comment]
> The video element does not have this behavior?

Not as far as I can tell from [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#try_it), no.  So we probably should do likewise.

Firefox doesn't seem to use Alt+Left Arrow and Alt-Right Arrow for changing the time on the above video embed, but Chrome does. So, not sure what the "correct" behaviour should be in that case.   Will look a bit more.
